cfgov/unprocessed/css/organisms/image-text-50-50-group.less

```diff
@@ -16,10 +16,18 @@
     - cfgov-molecules
 */
 .o-image-text-50-50-group {
+
+    margin-bottom: unit( @grid_gutter-width * -2 / @base-font-size-px, em );
```

Comment:
We can go with this for now, but this seems like it's bound to give us trouble later. Can you add a TODO to explore other layout options just so we know to come back to this?